### DISCUSSION TITLE
Sidebar question

### DISCUSSION THREAD

**Discussion by homa57** [NemesisRE/kiosk-mode#99] [Q&A]
Is it possible to hide other individual elements like the profile entry? I would like to hide the entries logbook & history in the sidebar.

**Comment by NemesisRE** [comment]
General that would be possible but the question is what the scope of the project is. Kiosk Mode was made to be used as a tablet interface with limited interaction and information.

With options like that we are more and more going in the direction of rights management, which is and should not be the scope of this project.

**Reply by homa57** [reply]
That is certainly true. Unfortunately, rights management is not mature enough in my opinion. I come from FHEM and am currently evaluating Home Assistant. In principle, I find the visual settings in Home Assistant more convenient. I mainly use KNX. However, more and more alternatives have been added recently ...
In a family with a very curious teenager, you still have to protect some things ;-)
But you are right, after two days he shows me that it goes to the other units via the LOG... but he has not yet discovered the URL /profile! 
It would just be practical and beautiful.
Thank you for your great project.

**Reply by elchininet** [reply]
Hi @homa57,
You can explore [card-mod](https://github.com/thomasloven/lovelace-card-mod), it has the level of personalisation that you are searching, You can achieve it using a [card-mod-theme](https://github.com/thomasloven/lovelace-card-mod/wiki/Card-mod-Themes).
Regards

**Reply by elchininet** [reply]
But if you want to hide only the logbook & history entry-points in the sidebar (the pages are still accesible through the URLs), follow these steps:

1. Enter in the profile page
2. Search for `Change the order and hide items from the sidebar`
3. Just remove those items from the sidebar and click on `Done` in the upper part of it

**Comment by elchininet** [comment]
Closing this discussion, if someone reaches here trying to hide items from the sidebar, try [custom-sidebar](https://github.com/elchininet/custom-sidebar).
